src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -155,9 +155,12 @@ Copyright (c) .NET Foundation. All rights reserved.
                    ResourceName="AspNetCoreUsesFrameworkReference" />
     
   </Target>
-  
-  <Target Name="ApplyImplicitVersions" BeforeTargets="_CheckForInvalidConfigurationAndPlatform;CollectPackageReferences"
-          DependsOnTargets="UpdateAspNetToFrameworkReference;CheckForImplicitPackageReferenceOverrides">
+
+  <Target Name="ApplyImplicitVersions" 
+          BeforeTargets="_CheckForInvalidConfigurationAndPlatform;CollectPackageReferences;ResolveFrameworkReferences"
```

Comment:
What's the reason for adding ResolveFrameworkReferences to the `BeforeTargets`?

Reply:
Because it adds package references

Reply:
They both run before CollectPackageReferences, but I want to go first here.

Reply:
Is there a reason it needs to go first?  I wasn't able to follow why it would matter.

Reply:
Because the Condition on '@(PackageReference)' != '' would not be true after ResolveFrameworkReferences runs. This is actually only an issue now while we're using PackageReference to simulate PackageDownload.

Reply:
So if the only PackageReferences we have are fake PackageDownloads, we shouldn't need to apply implicit versions or add the platforms package, right?

Reply:
right.